### DISCUSSION TITLE
crazyflie-lib-python fpv.py example

### DISCUSSION THREAD

**Discussion by 4perros** [bitcraze/discussions#380] [Q&A]
hi
i would like to know if it is possible to save the streaming video from the fpv.py example.
regards
rodrigo

**Comment by knmcguire** [comment]
Currently in the FPV example script does not have the a save options. There are several options what you can consider:
- Figure out how to save this image type in the script: https://github.com/bitcraze/crazyflie-lib-python/blob/8d82a769952d1b49de0d6779a2b59ba94036151a/examples/aideck/fpv.py#L189
- Use a screen recorder or obsstudio (did that for the video)
- Use the opencv script of the wifi streamer instead: https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#flash-wifi-example if you are only interested in the pictures. That one has a [--save option](https://github.com/bitcraze/aideck-gap8-examples/blob/d59e22c6bcb93b4fc23b90cee29e1341f5c355a2/examples/other/wifi-img-streamer/opencv-viewer.py#L50) which saves each image.
